### PR TITLE
Add ability to get targets and display them in the map

### PR DESCRIPTION
**Pull request by germanStabile**
#### Add ability to get targets and display them in the map

---

* [Trello Board](https://trello.com/b/YlTHCrRB/target-mvd-german-stabile)
* [Trello Card](https://trello.com/c/rURV5A7V/5-3-as-a-user-i-should-be-able-to-see-all-my-previously-created-targets)

---

#### Reviewers:

* @missx @aguscha333 @GAKINDUSTRIES @bernabe9 

---

#### Preview:
file

### CODE REVIEW

File: src/reducers/target.js [hunk 1]
```diff
@@ -26,10 +30,15 @@ const targetReducer = (state = initialState, action) => {
       const newState = state.set('isLoading', false);
       return newState.set('targetCoords', null);
     }
+    case GET_TARGETS_SUCCESS: {
+      const newState = state.set('isLoading', false);
```
Comment (aguscha333) [inline src/reducers/target.js]: This is just a personal preference but I like to be consistent and always set the loader at the end of the reducer case. It gives the cases more structure.

File: src/reducers/target.js [hunk 2]
```diff
@@ -26,10 +30,15 @@ const targetReducer = (state = initialState, action) => {
...
+      return newState.set('targets', action.targets);
```
Comment (aguscha333) [inline src/reducers/target.js]: Same here, even though you are getting only one prop from `action` we usually destructure it anyways. Makes it look more clean and organize when all of them look alike.

File: src/components/common/MapComponent/index.js [hunk 3]
```diff
@@ -85,6 +95,26 @@ class MapComponent extends React.Component {
         region={region}
         onRegionChangeComplete={this.onRegionChange}
       >
+        {targets &&
+          targets.map((target, index) => {
+            const key = `marker${index}`;
+            const { lat, lng } = target.target;
```
Comment (aguscha333) [inline src/components/common/MapComponent/index.js]: This is just a comment and a personal preference, you don't need to change it if you don't agree.

I don't like when objects contain a prop with the name of the variable.  With `target.target` it looks like `target` contains a `target` when they are not referring to the same thing.

In these cases I usually call the mapping object or the variable I am creating with the object `objectData`, in this case `targetData`.

If you do want to change it you could also change `topic` for `topicData` so you don't do `topic.topic` as well.

**Comment by germanStabile** [comment]
@aguscha333 @missx it took more than two months but comments addressed! 😂

**Comment by germanStabile** [comment]
any additional comments will be tackled in #6

**Pull request merged**